specs/latest/2.0/index.html

```diff
@@ -1498,28 +1624,54 @@ <h2 class="no-toc">Table of contents</h2>
         </p>
       </dd>
 
-      <dt class="idl-code">void texImage3D(GLenum target, GLint level, GLint internalformat, GLsizei width, GLsizei height, GLsizei depth, GLint border, GLenum format, GLenum type, ArrayBufferView? pixels)
+      <dt class="idl-code">
+        void texImage3D(GLenum target, GLint level, GLint internalformat, GLsizei width,
+                        GLsizei height, GLsizei depth, GLint border, GLenum format, GLenum type,
+                        ArrayBufferView srcData, GLuint srcOffset)
         <span class="gl-spec">(<a href="http://www.khronos.org/registry/gles/specs/3.0/es_spec_3.0.4.pdf#nameddest=section-3.8.3">OpenGL ES 3.0.4 &sect;3.8.4</a>, <a class="nonnormative" href="https://www.khronos.org/opengles/sdk/docs/man3/html/glTexImage3D.xhtml">man page</a>)</span>
       </dt>
       <dd>
         <p>Allocates and initializes the specified mipmap level of a three-dimensional or two-dimensional array texture. </p>
         <p>If a WebGLBuffer is bound to the <code>PIXEL_UNPACK_BUFFER</code> target, generates an <code>INVALID_OPERATION</code> error.</p>
-        <p>If <code>pixels</code> is null, a buffer of sufficient size initialized to 0 is passed. </p>
-        <p>If <code>pixels</code> is non-null but its size is less than what is required by the specified <em>width</em>, <em>height</em>, <em>depth</em>, <em>format</em>, <em>type</em>, and pixel storage parameters, generates an <code>INVALID_OPERATION</code> error. </p>
+        <p>If <code>srcData</code> is null, a buffer of sufficient size initialized to 0 is passed. </p>
```

Comment:
srcData can't be null any more. Actually, that presents a problem for applications wishing to initialize the texture to 0 without explicitly allocating a large zeroed buffer. Probably it should be made nullable again.